src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPClientFeatures.java

```diff
@@ -145,6 +147,54 @@ public boolean isCaseSensitive(@NotNull PsiFile file) {
         return false;
     }
 
+    /**

```

Comment:
Default accessor for language grammar-specific comment prefixes/suffixes and statement terminator characters. All are empty if unspecified.